src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -579,9 +583,21 @@ CLASS zcl_abapgit_object_common_aff IMPLEMENTATION.
           RECEIVING
             result = lv_json_as_xstring.
 
+        " ABAP Language Version should not be serialized, remove it from lv_json_as_xstring
+        lv_string = zcl_abapgit_convert=>xstring_to_string_utf8( iv_data = lv_json_as_xstring ).
+
+        lv_pattern = ',\n\s*"abapLanguageVersion":\s"(cloudDevelopment|keyUser)"'.
```

Comment:
Maybe, we can also simplify the regular expression to be more stable for future ABAP language versions and other whitespaces

```suggestion
        lv_pattern = ',\s*"abapLanguageVersion"\s*:\s*"[^"]+"'.
```